hubblestack/extmods/modules/pulsar.py

```diff
@@ -271,7 +271,10 @@ def process(configfile='salt://hubblestack_pulsar/hubblestack_pulsar_config.yaml
                     sub['checksum'] = __salt__['file.get_hash'](pathname, sum_type)
                     sub['checksum_type'] = sum_type
                 if config.get('stats', False):
-                    sub['stats'] = __salt__['file.stats'](pathname)
+                    if os.path.exists(pathname):
+                        sub['stats'] = __salt__['file.stats'](pathname)
+                    else:
```

Comment:
This else block seems unnecessary and os.path.exists(pathname) can be clubbed with upper if statement.

Reply:
Actually we need to set the value of sub['stats'] in all cases either with actual value or as {}.

Reply:
ok